### PR TITLE
Update readme for `Support Spring Boot`

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Add support for spring boot 2.1.x.
* Update doc of branch and release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 This repo is for Spring Boot Starters of Azure services. It helps Spring Boot developers to adopt Azure services.
 
 ### Support Spring Boot
-This repository supports both Spring Boot 1.5.x and 2.0.x. Please read [this document](https://github.com/Microsoft/azure-spring-boot/wiki/Spring-Boot-dependency-version-management) for branch mapping.
```
Comment (xscript) [inline README.md]: Can we also add badges for each of 2.0 starters? That will be more clear than just text here.

Comment (Incarnation-p-lee) [reply README.md]: Make sense, I search for a while for that badge in the morning but cannot tell how to play right now. I will try it later.

Comment (xscript) [reply README.md]: Found this from shields.io:
image

Comment (Incarnation-p-lee) [reply README.md]: OK. I will try it later.

**Pull request merged**